frontend/chat-plugin/src/airyRenderProps/AiryInputBar/index.tsx

```diff
@@ -2,21 +2,25 @@ import React, {ChangeEvent, FormEvent, KeyboardEvent, createRef, useEffect} from
 import style from './index.module.scss';
 import {cyInputbarTextarea, cyInputbarButton} from 'chat-plugin-handles';
 
-type Props = {
+type AiryInputBarProps = {
   sendMessage: (text: string) => void;
   messageString: string;
   setMessageString: (text: string) => void;
+  setNewConversation: React.Dispatch<React.SetStateAction<boolean>>;
 };
 
-const AiryInputBar = (props: Props) => {
+const AiryInputBar = (props: AiryInputBarProps) => {
   const textInputRef = createRef<HTMLTextAreaElement>();
   const dataCyButtonId = cyInputbarButton;
   const dataCyTextareaId = cyInputbarTextarea;
 
   useEffect(() => {
+    if (textInputRef.current === null) {
```

Comment:
As we discussed in our call, if you do this outside the component you don't have to do anything inside the component and all the logic stays at high level